src/parse.ts

```diff
@@ -71,6 +69,9 @@ export async function parse(parser: Parser, spectral: Spectral, asyncapi: Input,
   
     // unfreeze the object - Spectral makes resolved document "freezed" 
     const validatedDoc = copy(validated as Record<string, any>);
+
+    // Apply unique ids which are used as part of iterating between channels <-> operations <-> messages
+    applyUniqueIds(validatedDoc);
```

Comment:
Executed now after validation.What is the wrong thing in previous implementation?